### DISCUSSION TITLE
Where can I find documentation or Examples for animations?

### DISCUSSION THREAD

**Discussion by Aytharn** [fsprojects/Avalonia.FuncUI#357] [Q&A]
In my understanding I should create a list of Transitions and set the .Transitions property of a component I wish to animate with it.
But nowhere is written how.
The templates don't even look to be up to date (TodoApp)

**Comment by JaggerJo** [comment]
You might need to look at the C#/XAML and figure it out. Let me know when you have then we'll add a section to the docs.

It should be quite simple and just work.

**Reply by Aytharn** [reply]
What I tried by doing: 
```
let easeIn (control: Control) =
        let doubleTransition = DoubleTransition(
            Property = "Opacity",
            Easing = Easings.LinearEasing,
            Duration = TimeSpan.FromSeconds 0.2
        )
        control.Opacity <- 0.0
        control.Transitions <- [doubleTransition]

```
but it doesn't compile
